spec/features/notifications_spec.rb

```diff
@@ -0,0 +1,99 @@
+require "rails_helper"
+
+feature "Notifications", type: :feature do
+  let(:organisation_show_page) { PageObjects::Page::Organisations::OrganisationShow.new }
+  let(:notifications_index_page) { PageObjects::Page::Notifications::IndexPage.new }
+
+  let(:provider) { build :provider }
+  let(:access_request) { build :access_request }
+  # TODO: remove admin flag when we are ready to release to users
+  let(:user) { build :user, admin: true }
+
+  before do
+    stub_omniauth(user: user)
+    stub_api_v2_resource(provider)
+    stub_api_v2_resource(provider.recruitment_cycle)
+    stub_api_v2_resource_collection([access_request])
+  end
+
+  context "When the provider is not an accredited body" do
+    it "does not have the notifications link" do
+      when_i_visit_providers_page
+      then_i_should_not_see_notifications_link
+    end
+  end
+
+  context "When the provider is an accredited body" do
```

Comment:
Do we need a test case for not selecting an option?

Reply:
Yeah, I was thinking about it but I think the validation etc. will be done when we do the final integration with the API.